pkg/workloads/cortex/lib/storage/local.py

```diff
@@ -141,3 +141,13 @@ def download_and_unzip(self, key, local_dir):
         local_zip = os.path.join(local_dir, "zip.zip")
         self.download_file(key, local_zip)
         util.extract_zip(local_zip, delete_zip_file=True)
+
+    def list_objects(self, path):
+        objects = []
+        for root, dirs, files in os.walk(path):
+            for name in files:
+                objects.append(os.path.join(root, name))
```

Comment:
Does this return the path relative to the `path` argument? If not, maybe it should to match the s3 implementation